### PR TITLE
Initial skeleton app

### PR DESCRIPTION
**Pull request by grote**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    id 'dagger.hilt.android.plugin'
+    id 'org.jlleitschuh.gradle.ktlint' version '10.2.0'
+}
+
+android {
+    compileSdk 31
+
+    defaultConfig {
+        applicationId "org.onionshare.android"
+        minSdk 21
```
Comment (akwizgran) [inline app/build.gradle:14]: Pragmatic :-)

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStorePath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionSha256Sum=13bf8d3cf8eeeb5770d19741a59bde9bd966dd78d17f1bbad787a05ef19d1c2d
```
Comment (akwizgran) [inline gradle/wrapper/gradle-wrapper.properties:6]: :heart:

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">Onionshare</string>
```
Comment (akwizgran) [inline app/src/main/res/values/strings.xml:2]: Nitpick: on the website it's OnionShare

File: app/src/main/java/org/onionshare/android/ui/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        .fillMaxHeight()
+    ) {
+        Text(text = "Hello $name!", modifier = Modifier
+            .padding(all = Dp(16.0f))
+            .align(Alignment.Center)
+        )
+    }
+}
+
+@Preview(showBackground = true)
```
Comment (akwizgran) [inline app/src/main/java/org/onionshare/android/ui/MainActivity.kt:49]: We fear change...

**Review by akwizgran — APPROVED** [review]
Looks good, thanks for the tutorial :-)

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (grote) [reply app/src/main/res/values/strings.xml]: Ooops. Good catch. I changed that in all instances I could fine to get used to that capitalization.

**Pull request merged**